### PR TITLE
Payee 359 employer journey 002

### PR DESCRIPTION
**Pull request by mahowarth**
- [x] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by Jamie22Butler — APPROVED** [review]
LGTM 👍

File: app/views/employments/AddEmploymentPayrollNumberFormView.scala.html [hunk 1]
```diff
@@ -14,64 +14,87 @@
...
-    pageHeader = Some(header),
-    articleClasses = None
+    backLinkContent = Messages("tai.back-link.upper")
 ) {
+    <div class = "govuk-grid-row">
+        <div class = "govuk-grid-column-two-thirds">
+            @if(payrollNumberForm.errors.nonEmpty) {
+                @govukErrorSummary(ErrorSummary(errorList = payrollNumberForm.errors.asTextErrorLinks, title = Text(messages("tai.income.error.form.summary"))))
+            }
+            @header
```
Comment (masahmad) [inline app/views/employments/AddEmploymentPayrollNumberFormView.scala.html:49]: do we need header ? - As the H1 heading is displayed in the legend on these journeys with forms ?

File: test/views/html/employments/AddEmploymentPayrollNumberFormViewSpec.scala [hunk 2]
```diff
@@ -62,9 +62,9 @@ class AddEmploymentPayrollNumberFormViewSpec extends TaiViewSpec with FormValues
           "tai.addEmployment.employmentPayrollNumber.error.selectOption")
         val formWithErrors: Form[AddEmploymentPayrollNumberForm] = AddEmploymentPayrollNumberForm.form
           .withError(AddEmploymentPayrollNumberForm.PayrollNumberChoice, noPayrollNumberChooseError)
-        def view: Html = add_employment_payroll_number_form(formWithErrors, payrollNumberViewModel)
+        val view = add_employment_payroll_number_form(formWithErrors, payrollNumberViewModel)
```
Comment (brendanstraughan) [inline test/views/html/employments/AddEmploymentPayrollNumberFormViewSpec.scala:65]: just out of curiosity why was this changed to a val?

File: app/views/employments/AddEmploymentPayrollNumberFormView.scala.html [hunk 3]
```diff
@@ -14,64 +14,87 @@
...
+                ).withFormField(payrollNumberForm(AddEmploymentPayrollNumberForm.PayrollNumberChoice)))
 
-        <div id="payrollNumberEntry-container" class="@if(payrollNumberForm(AddEmploymentPayrollNumberForm.PayrollNumberChoice).value.getOrElse("") != AddEmploymentPayrollNumberForm.YesValue){js-hidden} data-payrollNumberEntry">
-            @input(
-                field = payrollNumberForm(AddEmploymentPayrollNumberForm.PayrollNumberEntry),
-                labelText = Messages("tai.addEmployment.employmentPayrollNumber.label"),
-                classes = Seq("form-control input--cleared"),
-                panel = true
-            )
+                @govukButton(Button(content = Text(messages("tai.continue"))))
```
Comment (brendanstraughan) [inline app/views/employments/AddEmploymentPayrollNumberFormView.scala.html:92]: does this need to say submit like it did before?

Comment (sergi-hmrc) [reply app/views/employments/AddEmploymentPayrollNumberFormView.scala.html:92]: so in the messages file: tai.continue == tai.submit  == "Continue"
imo being that both of them point to continue I'd use continue but again buttons are submitted... I'm happy either way

File: test/views/html/employments/AddEmploymentPayrollNumberFormViewSpec.scala (hunk 2 above)
Comment (sergi-hmrc) [reply test/views/html/employments/AddEmploymentPayrollNumberFormViewSpec.scala:65]: because it's only called once and it's not reused outside the scope of this test

File: app/views/employments/AddEmploymentPayrollNumberFormView.scala.html (hunk 1 above)
Comment (sergi-hmrc) [reply app/views/employments/AddEmploymentPayrollNumberFormView.scala.html:49]: the only problem I see is that this view has the header on top of the list, should the list go as part of the Fieldset?
